### PR TITLE
pyjade: move to own file and fix homepage

### PR DESCRIPTION
**Pull request by etu**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [X] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/python-modules/pyjade/default.nix [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{ stdenv, buildPythonPackage, fetchPypi, pyramid_mako, nose, django, jinja2
+, tornado, pyramid, Mako, six }:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
```
Comment (FRidh) [inline pkgs/development/python-modules/pyjade/default.nix]: The `name` attribute is added by `buildPython*` and should therefore be removed.

File: pkgs/development/python-modules/pyjade/default.nix [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    nosetests pyjade
+  '';
+  # No tests distributed. https://github.com/syrusakbary/pyjade/issues/262
+  doCheck = false;
+  meta = with stdenv.lib; {
+    description = "Jade syntax template adapter for Django, Jinja2, Mako and Tornado templates";
+    homepage    = "https://github.com/syrusakbary/pyjade";
+    license     = licenses.mit;
+    maintainers = with maintainers; [ nand0p ];
+    platforms   = platforms.all;
```
Comment (FRidh) [inline pkgs/development/python-modules/pyjade/default.nix]: this should not be set

File: pkgs/development/python-modules/pyjade/default.nix [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  version = "4.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1mycn5cc9cp4fb0i2vzgkkk6d0glnkbilggwb4i99i09vr0vg5cd";
+  };
+
+  buildInputs = [ pyramid_mako nose django jinja2 tornado pyramid Mako ];
+  propagatedBuildInputs = [ six ];
+  patchPhase = ''
```
Comment (FRidh) [inline pkgs/development/python-modules/pyjade/default.nix]: postPatch

**Review by FRidh — CHANGES_REQUESTED** [review]
Thanks for the initiative. Let's improve the expression while at it.

File: pkgs/development/python-modules/pyjade/default.nix (hunk 1 above)
Comment (etu) [reply pkgs/development/python-modules/pyjade/default.nix]: Fixed :+1:

File: pkgs/development/python-modules/pyjade/default.nix (hunk 2 above)
Comment (etu) [reply pkgs/development/python-modules/pyjade/default.nix]: Removed

File: pkgs/development/python-modules/pyjade/default.nix (hunk 3 above)
Comment (etu) [reply pkgs/development/python-modules/pyjade/default.nix]: Changed

**Pull request merged**